transaction-ballerina/src/transaction/tests/transaction_stmt.bal

```diff
@@ -512,3 +512,56 @@ function testIsolatedTransactionalAnonFunc() {
     }
     test:assertEquals(ss, "trxStarted -> within isolated transactional anon func -> trxEnded.");
 }
+
+@test:Config {
+}
+function testRollbackInElseStatement() returns error? {
+    string ss = "";
+    transaction {
+        ss = "trxStarted";
+        var x = commit;
```

Comment:
This should not be allowed as per spec. Once we do a commit, the strand comes out of transactional mode. And for a rollback to be allowed, the strand should be in transactional mode.
The following is allowed:
```ballerina
if(something) {
  commit;
} else {
  rollback;
}
```

Reply:
Fixed